### PR TITLE
Reorganize dev tools

### PR DESCRIPTION
**Pull request by janewang**
This is a preliminary PR to introduce pages geared towards each type of tools. 

Each section will be further expanded with details and documentations

### CODE REVIEW

File: docs/tools/developer-tools/IDEs.mdx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: IDEs
+description: Explore a suite of developer tools for Stellar, including anchor directory, demo wallet, Polaris, asset sandbox, block explorers, CLI, data indexers, GUIs, IDEs, network status, smart contract resources, and wallets.
```
Comment (willemneal) [inline docs/tools/developer-tools/IDEs.mdx:3]:
```suggestion
description: Explore a suite of developer tools for Stellar, including directory of anchors, demo wallet, Polaris, asset sandbox, block explorers, CLI, data indexers, GUIs, IDEs, network status, smart contract resources, and wallets.
```
This is just a nit. Feel free to ignore.

File: docs/tools/developer-tools/IDEs.mdx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+## IDEs
+
+### [Okashi](https://okashi.dev)
+
+A web GUI for building, deploying, and testing contracts.
+
+### [Sorobix](https://sorobix.vercel.app/)
+
+An IDE where you can write, deploy, and invoke contracts on the Stellar network using Soroban.
```
Comment (willemneal) [inline docs/tools/developer-tools/IDEs.mdx:16]: Isn't this also a web GUI?

**Review by willemneal — APPROVED** [review]
I think this is worth merging without a deep look over. I also don't think I have approval permissions.

File: docs/tools/developer-tools/IDEs.mdx (hunk 1 above)
Comment (janewang) [reply docs/tools/developer-tools/IDEs.mdx:3]: I will update the descriptions on these pages.

File: docs/tools/developer-tools/IDEs.mdx (hunk 2 above)
Comment (janewang) [reply docs/tools/developer-tools/IDEs.mdx:16]: Sure, adding

File: docs/tools/developer-tools/laboratory.mdx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+title: Laboratory
+description: Explore a suite of developer tools for Stellar, including Laboratory
```
Comment (quietbits) [inline docs/tools/developer-tools/laboratory.mdx:3]: Is this description accurate? This file has only Lab here.

**Review by quietbits — APPROVED** [review]

File: docs/tools/developer-tools/IDEs.mdx
Comment (briwylde08) [inline docs/tools/developer-tools/IDEs.mdx:1]: Quick nit! Can we remove the `## IDEs`? In the preview it looks like it's got two "IDEs" titles.

Comment (janewang) [reply docs/tools/developer-tools/IDEs.mdx:1]: Yup, good catch

File: docs/tools/developer-tools/laboratory.mdx (hunk 3 above)
Comment (janewang) [reply docs/tools/developer-tools/laboratory.mdx]: Updated the lab name and description

**Pull request merged**